.github/workflows/deploy_stage.yml

```diff
@@ -39,6 +39,9 @@ jobs:
           AIO_RUNTIME_NAMESPACE: ${{ secrets.AIO_RUNTIME_NAMESPACE_STAGE }}
           AIO_RUNTIME_AUTH: ${{ secrets.AIO_RUNTIME_AUTH_STAGE }}
           GROUP_CHECK_URL: ${{ vars.GROUP_CHECK_URL }}
+          ENABLE_PREVEW: ${{ vars.ENABLE_PREVEW }}
+          BULK_PREVIEW_CHECK_INTERVAL: ${{ vars.BULK_PREVIEW_CHECK_INTERVAL }}
+          MAX_BULK_PREVIEW_CHECKS: ${{ vars.MAX_BULK_PREVIEW_CHECKS }}
```

Comment:
The latter two are not setup as github vars yet. Do we have defaults for these? We should set these up in github nonetheless.

Reply:
Thanks Sunil! Added these 2 secrets in graybox-io settings.